bot/src/discord_commands/about.js

```diff
@@ -3,7 +3,7 @@ const monochromeVersion = require('monochrome-bot/package.json').version;
 
 const erisVersionString = `Eris v${erisVersion}`;
 const nameString = 'K33#5261';
-const otherDevsString = 'Alkhwarizm#7349, 💝 Amy 💝#0001';
+const otherDevsString = 'Alkhwarizm#7349, 💝 Amy 💝#0001, ernespa';
```

Comment:
If you want we could put `github.com/ernespa`. If that sounds better let me know and I will change it.

Reply:
I don't have any preference in that regard. I'm not an active GitHub user so anything is fine for me. Leave it as it is.